packages/contract-schema/spec/abi.spec.json

```diff
@@ -85,9 +88,12 @@
           "$ref": "#/definitions/StateMutability"
         },
         "constant": { "type": "boolean" },
-        "payable": { "type": "boolean", "default": false }
+        "payable": { "type": "boolean" }
```

Comment:
In testing the output on older solc versions, it seems that `"payable"` only appears when you specify the keyword in the Solidity source.

Commenting here cause this will need to change accordingly.

Reply:
Hm, so Truffle only supports back to 0.4.12. I tried 0.4.11 and it includes `payable` regardless of whether the constructor is payable or not, so I'm keen to ignore the case where `payable` isn't required.

@haltman-at could you resolve if you concur that we can ignore this issue?